api/modules/automations.js

```diff
@@ -30,20 +30,43 @@ const automations = module.exports = (() => {
     }
 
     const getOrganizationRepos = async (params) => {
-        const organizations = await getUserOrganizations({
-            auth_key: params.auth_key
-        })
-        const isOrganization = organizations[0].name != params.organizationName
         
+        const contributor = await db.models.Contributor.findOne({
+            where: {
+                github_id: params.accountId 
+            }
+        })
+
+        let organizations
+        let accountName
+        let isOrganization = false
+        if (contributor) {
+            const contributorSplit = contributor.github_handle.split(/\//)
+            accountName = contributorSplit[3]
+        } else {
+            organizations = await getUserOrganizations({
+                auth_key: params.auth_key
+            })
+        }
+
+        if (organizations) {
+            isOrganization = true
+            organizations.map((org, i) => {
```

Comment:
On the maps there's no need to create the `i` variable if it's not going to be used